neuralprophet/forecaster.py

```diff
@@ -3089,7 +3096,8 @@ def _reshape_raw_predictions_to_forecst_df(self, df, predicted, components):
                         yhat = np.concatenate(([np.NaN] * pad_before, forecast, [np.NaN] * pad_after))
                         if j == 0:  # temporary condition to add only the median component
                             name = f"{comp}{forecast_lag}"
-                            df_forecast[name] = yhat
+                            yhat_df = pd.Series(yhat, name=name).set_axis(df_forecast.index)
+                            df_forecast = pd.concat([df_forecast, yhat_df], axis=1, ignore_index=False)
```

Comment:
[for reviewer]: this cause pandas warnings with the previous implementation. Now fixed.